doc/introduction/circuits.rst

```diff
@@ -117,7 +117,8 @@ devices, the options are:
 
```

Comment:
Can't comment, but maybe the header in line 101 above should get the same formatting?

Reply:
Good call, have made the change :+1: